cz.ondrejkolin.Barcoder.yaml

```diff
@@ -0,0 +1,44 @@
+---
+id: cz.ondrejkolin.Barcoder
```

Comment:
`http://ondrejkolin.cz/` return a 403

Reply:
Nothing there still, but at least a valid 200 and working gitlab pages.